wiki/Article_styling_criteria/Formatting/fr.md

```diff
@@ -545,9 +545,9 @@ Voici un exemple des deux styles intégrés :
 Voici un exemple de style de référence :
 
 ```markdown
-[Modificateurs de jeu][lien vers les modificateurs de jeu]
+[Modificateurs de jeu][game mods link]
 
-[lien vers les modificateurs de jeu]: /wiki/Game_modifier
+[game mods link]: /wiki/Game_modifier
 ```
```

Comment:
Why modify other languages when the one concerned is ES?

What's more, link modifications like this don't show up on the osu!web side, and work correctly. I don't think it's forbidden to modify them.